src/repository/todo.repository/getToDos.ts

```diff
@@ -2,8 +2,8 @@ import fs from "fs";
 
 export default async function getToDos() {
```

Comment:
Return type missing.
Define the model/interface for a single Todo and use it as a return type here.